package.json

```diff
@@ -9,9 +9,7 @@
   "scripts": {
     "build": "pnpm run -r build",
     "format": "prettier --write .",
-    "format:check": "prettier --check .",
-    "lint": "eslint src/dashboard_ui",
-    "lint:fix": "pnpm lint --fix"
```

Comment:
Don't we have `eslint` configured for this repo?

Reply:
Not yet, we can add the NPM script back in once it's added